### DISCUSSION TITLE
Remove a package without trying to uninstall

### DISCUSSION THREAD

**Discussion by aaronliu0130** [ScoopInstaller/Scoop#5925] [Help and Questions] [answered]
It seems that I've uninstalled an np package without consulting scoop. As a result, I now have a ghost package which scoop cannot do anything on. How do I remove that package from scoop's knowledge?

**Comment by aaronliu0130** [comment]
@HUMORCE Since you've closed this, do you have a solution?

**Comment by HUMORCE** [comment, marked as answer]
Sorry, I have forgotten to attach it.

---

You can direct delete directory from `<ScoopRootDirecotry>\apps\<App>`, or `<ProgramData>\scoop\apps\<App>` if it is a global installation.
